tests/test_on_data/behavior/test_behavior_interfaces.py

```diff
@@ -365,6 +365,32 @@ def check_read_nwb(self, nwbfile_path: str):
             assert all(expected_pose_estimation_series_are_in_nwb_file)
 
 
+class TestDeepLabCutInterfaceNoConfigFile(DataInterfaceTestMixin):
+    data_interface_cls = DeepLabCutInterface
+    interface_kwargs = dict(
+        file_path=str(BEHAVIOR_DATA_PATH / "DLC" / "m3v1mp4DLC_resnet50_openfieldAug20shuffle1_30000.h5"),
+        config_file_path=None,
+        subject_name="ind1",
+    )
+    save_directory = OUTPUT_PATH
+
+    def check_read_nwb(self, nwbfile_path: str):
+        with NWBHDF5IO(path=nwbfile_path, mode="r", load_namespaces=True) as io:
+            nwbfile = io.read()
+            assert "behavior" in nwbfile.processing
+            processing_module_interfaces = nwbfile.processing["behavior"].data_interfaces
+            assert "PoseEstimation" in processing_module_interfaces
+
+            pose_estimation_series_in_nwb = processing_module_interfaces["PoseEstimation"].pose_estimation_series
+            expected_pose_estimation_series = ["ind1_leftear", "ind1_rightear", "ind1_snout", "ind1_tailbase"]
+
+            expected_pose_estimation_series_are_in_nwb_file = [
+                pose_estimation in pose_estimation_series_in_nwb for pose_estimation in expected_pose_estimation_series
+            ]
+
+            assert all(expected_pose_estimation_series_are_in_nwb_file)
```

Comment:
```suggestion
            assert all(
                pose_estimation in pose_estimation_series_in_nwb for pose_estimation in ["ind1_leftear", "ind1_rightear", "ind1_snout", "ind1_tailbase"]
            )
```

this would be clearer to me

Reply:
This is a pet peeve of mine that I think makes debuggin easier so I would like to explain myself.

The reason that I like avoid definitions inside of other scopes is that it gets on the way of the standard debuggin workflow.  With the current code If the test fails, I just set a debuggin point in line 390 (`pdb.set_trace()`) and then I can check immediatley what is the value of `expected_pose_estimation_series_are_in_nwb_file` and think about what failed. With the proposed change I would need to evaluate the expression again to inspect it.

Moreover, [before python 3.12 list comprehension is debugged as a separate frame](https://docs.python.org/3/whatsnew/3.12.html#pep-709-comprehension-inlining) so if I use the automated tools of my IDE I can't be in a frame that has access to both values of interest: the values inside the list comprehension and the context from which is called.

All of that said I think the first definition in line 385 can be eliminated to make this more concise but I like having all the defintions before the assertions/checks so I can debug everything in the same frame.

Reply:
ok, I rarely use pdb.set_trace, so I mostly go by readability, but I can see how it could get in the way for you. Feel free to ignore